yt_dlp/extractor/fifa.py

```diff
@@ -0,0 +1,106 @@
+            'description': 'md5:16f9f789f09960bfe7220fe67af31f34',
+            'ext': 'mp4',
+            'categories': ['FIFA Tournaments', 'Goal'],
+            'duration': 28,
+            'thumbnail': 'https://digitalhub.fifa.com/transform/f9301391-f8d9-48b5-823e-c093ac5e3e11/CG_MEN_1986_JOSIMAR',
+        },
+        'params': {'skip_download': 'm3u8'},
+    }]
+
+    def _real_extract(self, url):
+        video_id, locale = self._match_valid_url(url).group('id', 'locale')
+        webpage = self._download_webpage(url, video_id)
+
+        preconnect_link = self._search_regex(r'<link[^>]+rel\s*=\s*"preconnect"[^>]+href\s*=\s*"([^"]+)"', webpage, 'Preconnect Link')
+
+        json_data = self._download_json(
+            f'{preconnect_link}/video/GetVideoPlayerData/{video_id}', video_id,
+            'Downloading Video Player Data', query={'includeIdents': True, 'locale': locale})
+
+        video_details = self._download_json(f'{preconnect_link}/sections/videoDetails/{video_id}', video_id, 'Downloading Video Details', fatal=False)
+
+        preplay_parameters = self._download_json(
+            f'{preconnect_link}/video/GetVerizonPreplayParameters', video_id, 'Downloading Preplay Parameters', query={
+                'entryId': video_id,
+                'assetId': json_data['verizonAssetId'],
+                'useExternalId': False,
+                'requiresToken': json_data['requiresToken'],
+                'adConfig': 'fifaplusvideo',
+                'prerollAds': True,
+                'adVideoId': json_data['externalVerizonAssetId'],
+                'preIdentId': json_data['preIdentId'],
+                'postIdentId': json_data['postIdentId'],
+            })
+
+        cid = f'{json_data["preIdentId"]},{json_data["verizonAssetId"]},{json_data["postIdentId"]}'
+        content_data = self._download_json(
+            f'https://content.uplynk.com/preplay/{cid}/multiple.json', video_id, 'Downloading Content Data', query={
+                'v': preplay_parameters['preplayAPIVersion'],
+                'tc': preplay_parameters['tokenCheckAlgorithmVersion'],
+                'rn': preplay_parameters['randomNumber'],
+                'exp': preplay_parameters['tokenExpirationDate'],
+                'ct': preplay_parameters['contentType'],
+                'cid': cid,
+                'mbtracks': preplay_parameters['tracksAssetNumber'],
+                'ad': preplay_parameters['adConfiguration'],
+                'ad.preroll': int(preplay_parameters['adPreroll']),
+                'ad.cmsid': preplay_parameters['adCMSSourceId'],
+                'ad.vid': preplay_parameters['adSourceVideoID'],
+                'sig': preplay_parameters['signature'],
+            })
+
+        formats = self._extract_m3u8_formats(
+            content_data['playURL'], video_id, note='Downloading m3u8 Information')
+
+        return {
+            'id': video_id,
+            'title': json_data['title'],
```

Comment:
```suggestion
            'title': json_data.get('title'),
```

Reply:
I use [] because title is a mandatory metafield. 
https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#example-1

Why should I use .get instead?

Reply:
It is no longer strictly mandatory in yt-dlp. https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#:~:text=must%20also%20allow. I forgot to update that section of the docs